site/content/docs/5.0/examples/cover/cover.rtl.css

```diff
@@ -0,0 +1,88 @@
+/*
+ * Globals
+ */
+
+/* temp, until https://www.fontface.me/ combines these fonts into one family. */
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.woff") format("woff");
+}
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai/dubai.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai/dubai.woff") format("woff");
+}
```

Comment:
Same comment as for the pricing example: is this really needed?

Reply:
I'd rather not add such assets, so that's a hard no from me.

Reply:
I was going to use a single import statement for the fonts, however, even though they're the same family, their `font-family` where different (i.e. Dubai, Dubai-light, Dubai-heavy ...), which would make it unintuitive when using `<strong>` and `<small>` or the typed font sizes. I contacted the guy who maintains the CDN (https://www.fontface.me/) to fix this issue and he agreed to do it but I'm still waiting for him to publish the changes.

Reply:
The problem is we'd rather not use such CDNs. We only use Google Fonts for fonts. So, if we can skip adding this whole thing, that would be the best.

Reply:
I'll try and replace them with ones from Google Fonts.